src/Jackett.Common/Services/CacheService.cs

```diff
@@ -19,26 +19,23 @@ public void CacheRssResults(IIndexer indexer, IEnumerable<ReleaseInfo> releases)
         {
             lock (cache)
             {
-                var trackerCache = cache.Where(c => c.TrackerId == indexer.ID).FirstOrDefault();
+                var trackerCache = cache.FirstOrDefault(c => c.TrackerId == indexer.ID);
                 if (trackerCache == null)
                 {
-                    trackerCache = new TrackerCache();
-                    trackerCache.TrackerId = indexer.ID;
-                    trackerCache.TrackerName = indexer.DisplayName;
+                    trackerCache = new TrackerCache
+                    {
+                        TrackerId = indexer.ID,
+                        TrackerName = indexer.DisplayName
+                    };
                     cache.Add(trackerCache);
                 }
 
                 foreach (var release in releases.OrderByDescending(i => i.PublishDate))
                 {
-                    var existingItem = trackerCache.Results.Where(i => i.Result.Guid == release.Guid).FirstOrDefault();
-                    if (existingItem == null)
-                    {
-                        existingItem = new CachedResult();
-                        existingItem.Created = DateTime.Now;
+                    var existingItem = trackerCache.Results.FirstOrDefault(i => i.Result.Guid == release.Guid) ??
```

Comment:
the same as before, this code is not equivalent, you are calling `trackerCache.Results.Add(existingItem);` always...
please pay more attention. I can't walk behind you forever

Reply:
In large, sweeping, mostly automated refactoring like this, there are going to be things that fall through the cracks. I wouldn't be doing something like this alone because no matter how good someone is, or how good their tools, there will be mistakes. I do generally pay attention to what I'm doing, and this PR took me like 5 days just to review for the initial submission.

I do what I can, but I'm not perfect.